hostpath-operator.yaml

```diff
@@ -24,28 +24,11 @@ rules:
   resources: ["nodes", "nodes/proxy"]
   verbs: ["*"]
 - apiGroups: ["*"]
-  resources: ["namespaces", "services", "pods", "pods/exec", "deployments", "deployments/finalizers", "replicationcontrollers", "replicasets", "events", "endpoints", "configmaps", "secrets", "jobs", "cronjobs"]
```

Comment:
@prateekpandey14 Also limited the permissions for the hostpath operator. Can you check if I need to add any adiditional perms.

Reply:
looks good